src/graphnet/models/task/task.py

```diff
@@ -55,11 +60,12 @@ class Task(LightningModule):
     def nb_inputs(self) -> int:
         """Number of inputs assumed by task."""
 
+    @save_config
     def __init__(
         self,
         hidden_size: int,
         target_labels: Union[str, List[str]],
-        loss_function: LossFunction,
+        loss_function: "LossFunction",
```

Comment:
Is this intentional?

Reply:
Yes, this is to avoid a cyclic dependency caused by exposing `StandardModel` directly from `graphnet.models`. It has no practical effect, but is just a way to add type hinting for classes that are not directly imported.